charts/public-service/README.md

```diff
@@ -125,14 +125,14 @@ $ helm upgrade --install \
 ### Set up static IP address
 k8s by default reserves different IP addresses after each restart. In order to have a static one, you need to manually create it.
 This IP address will be later used to create a new load balancer and can be pointed from DNS.
-1. Navigate to `Dashboard -> Networking -> VPC Network -> External IP Addresses -> Reserve a static address`
+1. In GCP, navigate to [`Dashboard -> Networking -> VPC Network -> External IP Addresses -> Reserve a static address`](https://console.cloud.google.com/networking/addresses/add)
 1. Write a name for the static IP address
 1. Choose `(Global) IP address`
-1. Create the IP address
+1. Reserve the IP address
 1. Use the name of the static IP in `ingress.annotations: kubernetes.io/ingress.global-static-ip-name: 'your_ ip_address_name'`
 
 ### Show your certificates
-1. Navigate to `Network Services -> Load balancing -> advanced menu -> Certificates`
+1. In GCP, Navigate to [`Network Services -> Load balancing -> advanced menu -> Certificates`](https://console.cloud.google.com/net-services/loadbalancing/advanced/sslCertificates/list)
```

Comment:
i would recommend to use gcloud cmd since it's uch easier to copy/paste/adjust and execute it wihout having to click around.

```
gcloud compute ssl-certificates list --project=<project-id>
```

Reply:
https://github.com/commercetools/k8s-charts/pull/16/commits/2b963b14d4fd0cdd609f8150330c1b9e93969d0b